src/chainparams.cpp

```diff
@@ -472,6 +477,11 @@ class CTestNetParams : public CChainParams {
             /* nTxCount */ 178351,
             /* dTxRate  */ 0.03842042178237066
         };
+
```

Comment:
Add a new hardfork height to chain params, we won't use Eunos for this, call it FUpgradeHeight for now